packages/webdriver/src/utils.ts

```diff
@@ -204,7 +204,8 @@ export function getPrototype ({ isW3C, isChrome, isFirefox, isMobile, isSauce, i
          * only apply special commands when running tests using
          * Selenium Grid or Selenium Standalone server
          */
-        isSeleniumStandalone ? SeleniumProtocol : {}
+        isSeleniumStandalone ? SeleniumProtocol : {},
+        {} as Protocol
```

Comment:
Is this needed for the type conversion?
```suggestion
        isSeleniumStandalone ? SeleniumProtocol : {}
```

Reply:
It's the only way I was able to resolve this error: 
<img width="933" alt="Screen Shot 2023-01-02 at 2 55 45 AM" src="https://user-images.githubusercontent.com/29472529/210222233-144f933e-1409-4ba6-b82a-41ac52308f0c.png">